### ISSUE TITLE
Insufficient color contrast in the interactive examples

### ISSUE THREAD

**Issue by yarusome** [mdn/yari#8236]
### Summary

In the dark theme, the color of the `url` keyword is #2545a7 in the interactive examples, which makes the keyword hard to see against the background color #343434.

### URL

https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-image

### Reproduction steps

1. Navigate to the "Try it" [section](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-image#try_it) of the `list-style-image` MDN page.
2. Select the dark theme in the topright dropdown menu on the page.
3. Look at the first example in the section.

### Expected behavior

The keyword `url` be seen easily.

### Actual behavior

`url` is hard to see.

### Device

Desktop

### Browser

Firefox

### Browser version

Stable

### Operating system

Windows

### Screenshot

![sample](https://user-images.githubusercontent.com/97945148/219534310-6f2320ae-8d38-4bac-9e3e-33526b68ab0b.png)

### Anything else?

Contrast check [result](https://webaim.org/resources/linkcontrastchecker/?fcolor=FFFFFF&bcolor=343434&lcolor=2545A7):
![check](https://user-images.githubusercontent.com/97945148/219534810-ca646d64-6448-49bc-bb0f-96eb46b0edd2.png)

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).
